src/ui/public/vis/editors/default/agg_groups.js

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Could we please rename this file to `agg_group_names`? We often tend to have a high confusion around file names, like `agg_config` vs. `agg_configs`, so I would try to avoid that here.